### ISSUE TITLE
File block with a pdf upload becomes invalid in the editor when changing languages

### ISSUE THREAD

**Issue by talldan** [WordPress/gutenberg#43013]
### Description

Discovered in https://github.com/WordPress/gutenberg/pull/42745#discussion_r938429705.

The file block uses `__` in its `save` function. It's a static block so the internationalized output causes the block to be non-deterministic, resulting in validation errors.

Typically `save` output for static blocks can't be internationalized.

This could be a tricky one to solve, as removing the use of `__` could cause a lot of blocks added in the non-default language to become invalidated. Maybe a deprecation is possible though.

### Step-by-step reproduction instructions

To repro (in the post editor):

1. Upload a pdf to a file block
2. Publish the post
3. Change to a different language
4. Reload the post in the editor
5. Observe that the file block now has a validation error

I would recommend reproducing without the Gutenberg plugin active, as a lot of the translations are missing (at least for the dev version).

### Screenshots, screen recording, code snippet

![Screen Shot 2022-08-05 at 12 33 52 pm](https://user-images.githubusercontent.com/677833/183002177-197117ad-eb83-4691-977b-752239bc9644.png)

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by petitphp** [comment]
I successfully reproduced the issue with a WordPress 6.0.1 install without the Gutenberg plugin.

* I've inserted a file block in a new article, uploaded a pdf and save it,
* I went to the user profile and changed the site locale from EN to FR,
* I've reopened the article and observed that the file block encountered a validation error.

This error displayed in the web console :
```
Block validation: Block validation failed for `core/file`

Content generated by `save` function:

<div class="wp-block-file"><object [...] aria-label="Contenu embarqué lorem_ipsum."></object>[...]</div>

Content retrieved from post body:

<div class="wp-block-file"><object [...] aria-label="Embed of lorem_ipsum."></object>[...]</div>
```

The validation error did come from the localized aria-label of the object element changing with the locale.

**Comment by talldan** [comment]
This one was fixed by #43050
